# z-test

## Case 1 two tailed test

The average score of mathematics in the unified examination of the whole city is μ0 = 62, and the standard deviation is σ0 = 10.2. The average score of 90 students in a school is 68. Is the school's performance significantly different from the city's average. (take α = 0.05)


H0 : μ ＝ 62
<br/>
H1 : μ ≠ 62

μ0 = 62, σ0 = 10.2, $\bar{X}$ = 68, n = 90

Z = ${\bar{X}-μ0\over σ0 / \sqrt n}$ = ${68-62\over 10.2 / \sqrt 90}$



In [5]:
import math
Z = (68 - 62) / (10.2/math.sqrt(90))
print(Z)

5.580489988532435



α/2 = 0.025, look up table $Z_{α/2}$ = 1.96

Z = 5.58 > 1.96, so reject H0

******

## Case 2 one tailed test

Some people investigated the influence of early education on children's intellectual development. 70 children were randomly selected from the well-educated children to take the Wechsler children's intelligence test (μ0 = 100, σ0 = 15) results $\bar{X}$ = 103.3. Can we think that the intelligence of children with good early education is higher than the general level.

H0 : μ ≤ μ0
<br/>
H1 : μ > μ0

μ0 = 100, σ0 = 15, $\bar{X}$ = 103.3, n = 70

Z = ${\bar{X}-μ0\over σ0 / \sqrt n}$ = ${103.3-100\over 15 / \sqrt 70}$


In [7]:
import math
Z = (103.3 - 100) / (15/math.sqrt(70))
print(Z)

1.8406520583749646



α = 0.05, look up table $Z_{α}$ = 1.645

Z = 1.84 > 1.645, so reject H0